eks-deployments/README.md

```diff
@@ -28,6 +28,8 @@ aws ssm list-command-invocations --filter key=DocumentName,value=$domainjoinSSMd
 # If the group doesn't exist, it'll create a new one and then adds the instance to that AD group.
 # You can execute the ssmdomain join document on-demand on that instnace id.
 # To retry on failed instance (replace XXXXX with failed instance id)
+# If you follow the step "Create and join gMSA AD security group", the instance will be joined to the domain already.
```

Comment:
nit - i think you're implying that the instance will be domain joined already prior to this step? I don't know if this is confusing to customers going through the setup for the first time but try simplifying if possible.

Reply:
Agree, I ll rephrase this part.

Reply:
Please rephrase this.